app/views/devices_guidance/how_to_order_for_social_care.html.erb

```diff
@@ -0,0 +1,215 @@
+<% content_for :devices_service, true %>
+<% content_for :title, 'Order laptops and get internet access for care leavers' %>
+
+<%- content_for :before_content do %>
+  <% breadcrumbs([{ "Home" => root_path },
+                  "Get laptops",
+                 ]) %>
+<% end %>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Order laptops and get internet access for care leavers
+    </h1>
+    <p class="govuk-body">
+      This guidance explains which care leavers are eligible to receive laptops, and how local authorities (LAs) can place
+      orders through the Department for Education’s Get help with technology service.
+    </p>
+
+    <p class="govuk-body">
+      Care leavers cannot order directly.
+    </p>
+
+    <p class="govuk-body">
+      We recommend that you get set up to order as soon as possible to make sure you receive your preferred choice of laptops.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      Which care leavers are eligible for this support
+    </h2>
+
+    <p class="govuk-body">
+      DfE is providing these to support care leavers who:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>are aged between 16 and 25 years old</li>
+      <li>do not have access to a laptop or tablet from your LA, or any other source</li>
+    </ul>
+
+    <p class="govuk-body">
+      LAs can <%= govuk_link_to 'request 4G wireless routers for eligible care leavers', how_to_request_4g_wireless_routers_path %>
+      without fixed broadband too.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      How to get laptops and internet access
+    </h2>
+
+    <h3 class="govuk-heading-m">
+      1. Check your local authority’s laptop allocation
+    </h3>
+
+
+    <p class="govuk-body">
+      DfE has asked each LA to nominate one or more staff members to order laptops and request internet access for care leavers.
+    </p>
+
+    <p class="govuk-body">
+      If you’ve been nominated to order, we’ll email your LA’s estimated laptop allocation to you. If you have not received
+      this email, <%= govuk_link_to 'contact us', support_ticket_path %>.
+    </p>
+
+    <p class="govuk-body">
+      Your estimate is based on your individual care leaver population since August 2020.
```

Comment:
Please change to: Allocation estimates are based on each local authority's share of the overall care leaver population.